cli/src/test/java/com/box/l10n/mojito/cli/command/checks/CheckerUtilsTest.java

```diff
@@ -0,0 +1,76 @@
+package com.box.l10n.mojito.cli.command.checks;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class CheckerUtilsTest {
+
+  @Test
+  void weShouldBeAbleTo_getWordsInString_providingASingleWordAsInput() {
+    List<String> result = CheckerUtils.getWordsInString("Hello");
+    List<String> expectedResult = Collections.singletonList("Hello");
+    assertEquals(expectedResult, result);
+  }
+
+  @Test
+  void weShouldBeAbleTo_getWordsInString_providingMultipleWordsAsInput() {
+    List<String> result = CheckerUtils.getWordsInString("Hello, how can I help you today?");
+    List<String> expectedResult = Arrays.asList("Hello", "how", "can", "I", "help", "you", "today");
```

Comment:
I just think about that now, but we also have assertJ library that makes the test a bit better, https://www.javadoc.io/doc/org.assertj/assertj-core/latest/org/assertj/core/api/AbstractIterableAssert.html#containsExactly(ELEMENT...)

Reply:
it really reads better. Done, thank you for the suggestion

Reply:
and thank you for providing an example @tedyoung :)